src/app/view-banker/view-banker.component.css

```diff
@@ -0,0 +1,216 @@
+.Web_View-Banker {
+  position: fixed;
+  width: 1440px;
+  height: 1024px;
+  background-color: #f2f2f2;
+  margin-left: -8px;
+  margin-top: -8px;
+}
+
+.nedbank-header {
+  width: 1440px;
+  height: 80px;
+  background-color: #006341;
+}
+
+.Large---Desktop-Tablet {
+  position: absolute;
```

Comment:
1.Give class name meaningful and correctly "---" this doesn't look good
2. In all the files please follow camelCase and give names correctly